keycloak-service-module/src/main/java/org/aerogear/mobile/keycloak_service_module/KeyCloakService.java

```diff
@@ -17,34 +19,37 @@
 
     private final Context appContext;
     private KeyCloakConfig config;
+    private String serverUrl;
+    private String clientId;
+    private String audience;
+    private String grantType;
+    private String subjectTokenType;
+    private String requestedTokenType;
+    private String realm;
+    private MobileCore core;
 
     public KeyCloakService(@NonNull Context appContext) {
         this.appContext = appContext.getApplicationContext();
     }
 
-
     /**
-     * Loads keycloak.json from R.raw.keycloak
+     * Exchanges the google id token and configures the KeyCloakService to serve requests
```

Comment:
Google? Or Keycloak? This method is empty, without a TODO, should we include it in the commit anyway?

Reply:
This is a test class that is going away in favor of IDM soon.  We shouldn't waste time on it.

Reply:
@josemigallas this method is removed in PR 17 https://github.com/secondsun/aerogear-android-sdk/blob/2c526fda60670ea761f27201b42058b2802a1730/keycloak-service-module/src/main/java/org/aerogear/mobile/keycloak_service_module/KeyCloakService.java